docs/conf.py

```diff
@@ -204,6 +204,14 @@
     "https://towardsdatascience.com/the-importance-of-layered-thinking-in-data-engineering-a09f685edc71",
     "https://medium.com/quantumblack/beyond-the-notebook-and-into-the-data-science-framework-revolution-a7fd364ab9c4",
     "https://www.java.com/en/download/help/download_options.html",  # "403 Client Error: Forbidden for url"
+
+    # Number section removal, temporarily ignore until 0.18.0 release
```

Comment:
@AntonyMilneQB You were right there were many unneeded links here.

Reply:
Cool, thank you!